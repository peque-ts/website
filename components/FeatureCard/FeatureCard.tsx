import React from 'react';

export const FeatureCard: React.VFC = () => {
  return <p>FeatureCard</p>;
};
