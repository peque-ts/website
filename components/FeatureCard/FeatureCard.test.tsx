import { render } from '@testing-library/react';

import { FeatureCard } from './FeatureCard';

describe('FeatureCard', () => {
  it('should render', () => {
    const { asFragment } = render(<FeatureCard />);

    expect(asFragment()).toMatchSnapshot();
  });
});
