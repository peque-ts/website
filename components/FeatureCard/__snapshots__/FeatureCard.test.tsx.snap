// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeatureCard should render 1`] = `
<DocumentFragment>
  <p>
    FeatureCard
  </p>
</DocumentFragment>
`;
